app/forms/hyrax/forms/resource_form.rb

```diff
@@ -53,16 +53,15 @@ class ResourceForm < Hyrax::ChangeSet # rubocop:disable Metrics/ClassLength
       #   we want to move away from application side lock validation and rely
       #   on the adapter/database features instead.
       LockKeyPopulator = lambda do |_options|
-        self.version =
-          case Hyrax.metadata_adapter
-          when Wings::Valkyrie::MetadataAdapter
+        if Hyrax.config.disable_wings || !Hyrax.metadata_adapter.is_a?(Wings::Valkyrie::MetadataAdapter)
```

Comment:
The `case` statement didn't allow for checking `disable_wings` before trying to use the `Wings` constant.  The change is a simple rewrite of the `case` into an `if` to allow for the more complex check.  The code that is executed for each case is the same.